src/main/java/project/anemonebot/anemone/AnemoneController.java

```diff
@@ -0,0 +1,42 @@
+package project.anemonebot.anemone;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Controller-class for incoming HTTP-calls to Anemone endpoints.
+ */
+
+@RestController(value = "anemone/")
+public class AnemoneController {
+
+
+    private static final String template = "Hello, %s!";
+    private final AtomicLong counter = new AtomicLong();
+
+
+    /**
+     * Simple Endpoint for testing connection to micro-service.
```

Comment:
Vi bör nog ändra detta till "Endpoint connection to micro-service."  Istället för att nämna simple + testing.
Sån java dokumentation hör nog till testning i test foldern.